components-core/src/main/java/com/adyen/checkout/components/base/lifecycle/ViewModelExt.kt

```diff
@@ -38,3 +38,11 @@ inline fun <reified ViewModelT : ViewModel> viewModelFactory(
         }
     }
 }
+
+fun <T : ViewModel> getWithKey(provider: ViewModelProvider, key: String?, modelClass: Class<T>): T {
```

Comment:
This could be `fun <T : ViewModel> ViewModelProvider.get(key: String?, modelClass: Class<T>) : T`, it will look just like the default methods provided by androidx while also not clashing with their signatures. WDYT?

Reply:
good idea